libs/vkd3d/vkd3d_shaders.h

```diff
@@ -845,4 +845,293 @@ void main() {
+    0x00000000,0x00060006,0x00000011,0x00000000,0x505f6c67,0x7469736f,0x006e6f69,0x00070006,
+    0x00000011,0x00000001,0x505f6c67,0x746e696f,0x657a6953,0x00000000,0x00070006,0x00000011,
+    0x00000002,0x435f6c67,0x4470696c,0x61747369,0x0065636e,0x00070006,0x00000011,0x00000003,
+    0x435f6c67,0x446c6c75,0x61747369,0x0065636e,0x00030005,0x00000013,0x00000000,0x00060005,
+    0x00000015,0x565f6c67,0x65747265,0x646e4978,0x00007865,0x00040047,0x00000008,0x0000001e,
+    0x00000000,0x00040047,0x0000000a,0x0000000b,0x0000002b,0x00050048,0x00000011,0x00000000,
+    0x0000000b,0x00000000,0x00050048,0x00000011,0x00000001,0x0000000b,0x00000001,0x00050048,
+    0x00000011,0x00000002,0x0000000b,0x00000003,0x00050048,0x00000011,0x00000003,0x0000000b,
+    0x00000004,0x00030047,0x00000011,0x00000002,0x00040047,0x00000015,0x0000000b,0x0000002a,
+    0x00020013,0x00000002,0x00030021,0x00000003,0x00000002,0x00040015,0x00000006,0x00000020,
+    0x00000001,0x00040020,0x00000007,0x00000003,0x00000006,0x0004003b,0x00000007,0x00000008,
+    0x00000003,0x00040020,0x00000009,0x00000001,0x00000006,0x0004003b,0x00000009,0x0000000a,
+    0x00000001,0x00030016,0x0000000c,0x00000020,0x00040017,0x0000000d,0x0000000c,0x00000004,
+    0x00040015,0x0000000e,0x00000020,0x00000000,0x0004002b,0x0000000e,0x0000000f,0x00000001,
+    0x0004001c,0x00000010,0x0000000c,0x0000000f,0x0006001e,0x00000011,0x0000000d,0x0000000c,
+    0x00000010,0x00000010,0x00040020,0x00000012,0x00000003,0x00000011,0x0004003b,0x00000012,
+    0x00000013,0x00000003,0x0004002b,0x00000006,0x00000014,0x00000000,0x0004003b,0x00000009,
+    0x00000015,0x00000001,0x0004002b,0x00000006,0x00000017,0x00000001,0x0004002b,0x0000000c,
+    0x0000001a,0x40800000,0x0004002b,0x0000000c,0x0000001c,0x3f800000,0x0004002b,0x00000006,
+    0x0000001f,0x00000002,0x0004002b,0x0000000c,0x00000022,0x40000000,0x0004002b,0x0000000c,
+    0x00000025,0x00000000,0x00040020,0x00000027,0x00000003,0x0000000d,0x00050036,0x00000002,
+    0x00000004,0x00000000,0x00000003,0x000200f8,0x00000005,0x0004003d,0x00000006,0x0000000b,
+    0x0000000a,0x0003003e,0x00000008,0x0000000b,0x0004003d,0x00000006,0x00000016,0x00000015,
+    0x000500c7,0x00000006,0x00000018,0x00000016,0x00000017,0x0004006f,0x0000000c,0x00000019,
+    0x00000018,0x00050085,0x0000000c,0x0000001b,0x00000019,0x0000001a,0x00050083,0x0000000c,
+    0x0000001d,0x0000001b,0x0000001c,0x0004003d,0x00000006,0x0000001e,0x00000015,0x000500c7,
+    0x00000006,0x00000020,0x0000001e,0x0000001f,0x0004006f,0x0000000c,0x00000021,0x00000020,
+    0x00050085,0x0000000c,0x00000023,0x00000021,0x00000022,0x00050083,0x0000000c,0x00000024,
+    0x00000023,0x0000001c,0x00070050,0x0000000d,0x00000026,0x0000001d,0x00000024,0x00000025,
+    0x0000001c,0x00050041,0x00000027,0x00000028,0x00000013,0x00000014,0x0003003e,0x00000028,
+    0x00000026,0x000100fd,0x00010038
+};
+
+const uint32_t gs_fullscreen_spv[] = {
+#if 0
+#version 450
+
+layout(triangles) in;
```

Comment:
Is GS really needed?

Reply:
We need it to support copying multiple layers at once.

Actually, good point, I think the draw call only does one layer at a time right now.